blog/community-update-wasmcloud-incubation-process/index.md

```diff
@@ -0,0 +1,24 @@
+
+<!--truncate-->
+
+The updated process, detailed [here](https://github.com/cncf/toc/blob/main/process/README.md#how-to-apply-to-move-levels), brings several upsides we wanted to provide as positive feedback. For one, it introduces a clearer, more structured template and process for projects to move through different levels of maturity within the CNCF. Previously, applicants had to fill out a separate document in addition to a PR, which included many repetitive sections and lacked clarity. It was also unclear exactly who was supposed to help move things along and in what timeframe. With the new issue template, many of these redundancies have been removed and simplified, minimizing confusion and making the overall due diligence process more transparent. This is great for maintaining high standards and building trust within the CNCF ecosystem. Additionally, each step of the process is explicitly layed out in the new document, making it easier for new applicants to understand the process and what it entails.
+
+However, there are still some redundant sections in the issue template for applying to incubation, especially around how the project fits into the cloud native space, which can make the process more time-consuming. Additionally, in the template there is a lack of clarity about the exact standards projects need to meet for some of the points, although we feel confident we have met them for wasmCloud. Some of the data points required for due diligence could potentially be automated using CNCF DevStats, which would reduce the burden on both the TOC and project maintainers, but this is more of a nit than a serious bother. Streamlining these small downsides could help make the process more efficient without compromising the thoroughness of the evaluations.
+
+Despite the extra time required, we’re incredibly excited about the direction we’re heading. The additional scrutiny and steps that come with incubating mean that once we achieve this status, we’ll be in a prime position to offer even greater value to our users and contributors. It’s a journey that not only validates the hard work we’ve put in but also sets us up for long-term success within the CNCF landscape while also supporting the community efforts to improve the incubation process.
+
+We'll finish this out with a huge thank you to the CNCF TOC for their efforts in improving the incubation process. Their dedication to enhancing the system ensures that projects like wasmCloud can continue to grow and contribute meaningfully to the cloud native community. We’re ready for incubation, eager to move forward, and so excited to take our amazing wasmCloud community with us on the next phase of our journey. Stay tuned for more updates!
```

Comment:
The natural response for readers will be to ask about timeframe; do we have an inkling?

Reply:
We don't yet, so I didn't want to insinuate anything